src/mindMapPreviewWebview.ts

```diff
@@ -18,9 +19,15 @@ class MindMapPreview {
     }
 
     configureDisposables(disposables: vscode.Disposable[]) {
-        disposables.push(vscode.workspace.onDidChangeTextDocument(() => {
-            this.updatePreview();
-        }));
+        let debounceFunc = utils.debounce(
+            this,
+            () => {
+                this.updatePreview.call(this, this.isFromFile);
+            },
+            2000
+        );
+
+        disposables.push(vscode.workspace.onDidChangeTextDocument(debounceFunc));

```

Comment:
This function is designed to not declare any variables. Please change the code style to this:

```suggestion
        disposables.push(vscode.workspace.onDidChangeTextDocument(utils.debounce(
                this,
                () => { this.updatePreview.call(this, this.isFromFile);  },
                2000 //please describe what does this 2000 mean.
        )));
```

Reply:
原来的代码，没有“节流”，即使敲下一个字符，也会重新渲染脑图，有点浪费性能；
添加“节流”，现在的表现是：用户停止输入后的2秒后，才重新渲染脑图。

Reply:
好的，请您使用
```typescript
2000 // Throttle 2000ms before rerending.
```
进行注释，谢谢。

另外个人还是希望这个函数内不要新增变量，你看一下我 suggest 的代码能不能跑，如果能的话就不要声明新的变量了。